docs/guides/getting-started.md

```diff
@@ -52,8 +52,8 @@ Next, we’ll create a `tokens.json` file in the root of our project (or `tokens
       }
     },
     "semantic": {
-      "action": "{color.base.blue.5}",
-      "textColor": "{color.base.gray.9}"
+      "action": {"$value": "{color.base.blue.5}"},
+      "textColor": {"$value": "{color.base.gray.9}"}
```

Comment:
🙏 Could you update the `YAML` example below as well?

Reply:
Good point, updated it here https://github.com/drwpow/cobalt-ui/pull/173/commits/a5bebb504bfc0229921662fac4b7f2df67844bf5. The test suite foremost covers the JSON input so keeping it there as is.